src/audio_sdl.cpp

```diff
@@ -180,7 +197,13 @@ SdlAudio::SdlAudio() :
 	if (Mix_OpenAudioDevice(frequency, MIX_DEFAULT_FORMAT, MIX_DEFAULT_CHANNELS, 2048,
 		NULL, SDL_AUDIO_ALLOW_FREQUENCY_CHANGE) < 0)
 #else
-	if (Mix_OpenAudio(frequency, MIX_DEFAULT_FORMAT, MIX_DEFAULT_CHANNELS, 2048) < 0)
+#  ifdef WORDS_BIGENDIAN
+	int format = AUDIO_S16MSB;
```

Comment:
At least for SDL Wii `MIX_DEFAULT_FORMAT = AUDIO_S16SYS = AUDIO_S16MSB`.
This should also be true for all "serious" SDL ports.